config/terraform.spc

```diff
@@ -2,8 +2,8 @@ connection "terraform" {
   plugin = "terraform"
 
   # Paths is a list of locations to search for Terraform configuration files.
-  # Wildcards are supported per https://golang.org/pkg/path/filepath/#Match
+  # Wildcard based searches are supported.
   # Exact file paths can have any name. Wildcard based matches must have an
-  # extension of .tf (case insensitive).
-  # paths = [ "/path/to/dir/*", "/path/to/exact/custom.tf" ]
+  # extension of .tf (case insensitive). Default set to current working directory.
```

Comment:
Can you include a few examples here like we do in https://github.com/turbot/steampipe-plugin-prometheus/blob/main/config/prometheus.spc#L10-L13 that highlight:
- Matching 2 exact filenames
- Scanning top level directory with `*.tf`
- Scanning recursively with `**/*.tf`